public/app/features/trails/ActionTabs/MetricOverviewScene.tsx

```diff
@@ -0,0 +1,154 @@
+    const variable = sceneGraph.lookupVariable(VAR_GROUP_BY, this)!;
+    if (!(variable instanceof QueryVariable)) {
+      throw new Error('Group by variable not found');
+    }
+
+    return variable;
+  }
+
+  private _onActivate() {
+    this.updateMetadata();
+  }
+
+  private onReferencedVariableValueChanged() {
+    this.updateMetadata();
+  }
+
+  private async updateMetadata() {
+    const ds = await getDatasourceSrv().get(VAR_DATASOURCE_EXPR, { __sceneObject: { value: this } });
+
+    const languageProvider: PrometheusLanguageProvider = ds.languageProvider;
+
+    if (!languageProvider) {
+      return;
+    }
+
+    const metricScene = getMetricSceneFor(this);
+    const metric = metricScene.state.metric;
+
+    if (languageProvider.metricsMetadata) {
+      this.setState({ metadata: languageProvider.metricsMetadata[metric] });
+      return;
+    }
+
+    await languageProvider.start();
+
+    this.setState({ metadata: languageProvider.metricsMetadata?.[metric] });
+  }
+
+  public static Component = ({ model }: SceneComponentProps<MetricOverviewScene>) => {
+    const { metadata } = model.useState();
+    const styles = useStyles2(getStyles);
+    const variable = model.getVariable();
+    const { loading } = variable.useState();
+    const labelOptions = getLabelOptions(model, variable).filter((l) => l.value !== ALL_VARIABLE_VALUE);
+
+    return (
+      <Stack gap={6}>
+        {loading ? (
+          <div>Loading...</div>
+        ) : (
+          <>
+            <Stack direction="column" gap={0.5}>
+              <div className={styles.label}>Description</div>
+              {metadata?.help ? <div>{metadata?.help}</div> : <i>No description available</i>}
+            </Stack>
+            <Stack direction="column" gap={0.5}>
+              <div className={styles.label}>Type</div>
+              {metadata?.type ? <div>{metadata?.type}</div> : <i>Unknown</i>}
+            </Stack>
+            <Stack direction="column" gap={0.5}>
+              <div className={styles.label}>Unit</div>
+              {metadata?.unit ? <div>{metadata?.unit}</div> : <i>Unknown</i>}
+            </Stack>
+            <Stack direction="column" gap={0.5}>
+              <div className={styles.label}>Labels</div>
+              {labelOptions.map((l) => (
+                <TextLink
+                  key={l.label}
+                  href={sceneGraph.interpolate(
+                    model,
+                    `/data-trails/trail$\{__url.params:exclude:actionView,var-groupby}&actionView=breakdown&var-groupby=${encodeURIComponent(
+                      l.value!
+                    )}`
+                  )}
+                  title="View breakdown"
+                >
+                  {l.label!}
+                </TextLink>
+              ))}
+            </Stack>
+          </>
+        )}
+      </Stack>
+    );
+  };
+}
+
+function getStyles(theme: GrafanaTheme2) {
+  return {
+    label: css({
+      fontWeight: 'bold',
```

Comment:
Should this be `'medium'` as per:
https://github.com/grafana/grafana/pull/80216/files#r1448674972